pkgs/by-name/cl/clamtk/package.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, perlPackages
+, fetchFromGitHub
+, makeWrapper
+, gobject-introspection
+, perl
+, clamav
+}:
+let
+  clamavpath = lib.makeBinPath [ clamav ];
+in
+perlPackages.buildPerlPackage rec {
+  pname = "clamtk";
+  version = "6.16";
+
+  src = fetchFromGitHub {
+    owner = "dave-theunsub";
+    repo = "clamtk";
+    rev = "v${version}";
+    hash = "sha256-o6OaXOXLykTUuF/taKnEhZRV04/3nlU5aNY05ANr1Ko=";
+  };
+
+  nativeBuildInputs = [ makeWrapper gobject-introspection ];
+  buildInputs = [ perl clamav ];
+  propagatedBuildInputs = with perlPackages; [ Glib LWP LWPProtocolHttps TextCSV JSON LocaleGettext Gtk3 ];
+
+  preConfigure = "touch Makefile.PL";
+  doCheck = false;
+  outputs = [ "out" "man" ];
+
+  postPatch = ''
+    # Set the correct paths for clamav programs
+    sed -i lib/App.pm -e  "93s@.*@        : \'${clamavpath}/freshclam\';@"
+    sed -i lib/App.pm -e "114s@.*@        : \'${clamavpath}/sigtool\';@"
+    sed -i lib/App.pm -e "123s@.*@        : \'${clamavpath}/clamscan\';@"
+    sed -i lib/App.pm -e "134s@.*@        : \'${clamavpath}/clamdscan\';@"
+
+    # Icons path
+    sed -i lib/App.pm -e "s@path->{ images } =.*@path->{ images } = '$out/share/pixmaps';@"
```

Comment:
Same as above. Just substitute the path. `/usr/share/pixmaps` to `$out/share/pixmaps`.